examples/vanilla/advanced.html

```diff
@@ -23,6 +23,21 @@
         display: none;
       }
 
+      video::-webkit-media-text-track-container {
+        transition: transform .25s;
+      }
+
+      media-controller:is([media-paused],:not([user-inactive])) video::-webkit-media-text-track-container {
+        transform: translateY(-50px);
+      }
+
+      video::-webkit-media-text-track-display {
```

Comment:
I think generally, I wouldn't recommend doing this. Particularly if users make changes to how they want their captions displayed. Chrome respects the macOS settings.


without these changes
![Screen Shot 2023-04-12 at 15 46 15](https://user-images.githubusercontent.com/535884/231568404-1888ca21-e420-41e3-95f3-c334cb196c08.png)

with these changes
![Screen Shot 2023-04-12 at 15 46 09](https://user-images.githubusercontent.com/535884/231568408-7573a87b-4c75-4516-8b2d-05e93512d5c2.png)

having the transform translateY is good, though.